portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/accessibility/Accessibility.testcase

```diff
@@ -21,6 +22,7 @@ definition {
 	}
 
 	@description = "Verify poshi function AssertAccessible passes atleast one portal page based on WCAG 2.0 AA standards"
+	@ignore = "Test Stub"
```

Comment:
```suggestion
	@ignore = "true"
```

Reply:
Yeah

Reply:
Hey @andreluis2022 could you take a look this?